### PR TITLE
Added timestamp, level and message parsing mechanism for opensearch logs

### PR DESCRIPTION
**Pull request by absBitm**
# Description

This includes parsing vital fields from open search logs i.e es-data, es-ingest and es-master.

Fixes VZ-5265

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml [hunk 1]
```diff
@@ -255,6 +255,29 @@ data:
       </parse>
     </filter>
 
+    # filter to parse opensearch log files which includes es-master, es-data, es-ingest
+    <filter kubernetes.**vmi-system-es-**verrazzano-system_es-**>
```
Comment (bobdonat) [inline platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml:259]: does this handle es-data, es-ingest, and es-master logs

**Comment by bobdonat** [comment]
should "System Logging Suite" ATs be updated?

File: platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml (hunk 1 above)
Comment (absBitm) [reply platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml:259]: yes

File: platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml [hunk 2]
```diff
@@ -255,6 +255,29 @@ data:
...
+          format /^\[(?<logtime>\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2},\d{3})\]\[(?<level>.*?)\]\[.*vmi-system-es-.*?\]\s(?<message>.*?)$/
+          time_key logtime
+          time_format %Y-%m-%dT%H:%M:%S,%N
+        </pattern>
+        <pattern>
+          format none
+        </pattern>
+      </parse>
+    </filter>
+
```
[the remark this answers is not in the document]
Comment (absBitm) [reply platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml:280]: one test function is already there "validateOpenSearchLogs()" in file [tests/e2e/logging/system/system_logging_test.go](https://github.com/verrazzano/verrazzano/pull/2739#diff-019d6a92371f19f132fa0047a67bc583b2bf74a80dc86274e3c2dd4871faade0)

[the remark this answers is not in the document]
Comment (bobdonat) [reply platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml:280]: but that does not validate the es-data, es-master, and es-ingest logs @kminder

**Pull request merged**